records_mover/creds/base_creds.py

```diff
@@ -13,6 +13,12 @@
 
 logger = logging.getLogger(__name__)
 
+_GSHEETS_SCOPES = ('https://www.googleapis.com/auth/spreadsheets',)
+
+_GCS_SCOPES = ('https://www.googleapis.com/auth/devstorage.full_control',
+               'https://www.googleapis.com/auth/devstorage.read_only',
+               'https://www.googleapis.com/auth/devstorage.read_write')
```

Comment:
Another more minor thing this PR fixes is that it limits the scope of the creds we pull when we pull from the default GCP creds store.

(The default GCP creds store is what gets populated when you either set the env variable that points to service account JSON, or you log in with `gcloud auth application-default login`)